praw/models/reddit/emoji.py

```diff
@@ -127,11 +127,27 @@ def __iter__(self):
                 self._reddit, self.subreddit, emoji_name, _data=emoji_data
             )
 
-    def add(self, name, image_path):
+    def add(
+        self,
+        name,
+        image_path,
+        post_flair=True,
+        user_flair=True,
+        mod_only=False
+    ):
         """Add an emoji to this subreddit.
 
         :param name: The name of the emoji
         :param image_path: A path to a jpeg or png image.
+        :param post_flair: A ``bool`` value. If ``False``, the emoji will not
+            be available in the emoji selection for post flair.
+            (default: ``True``).
+        :param user_flair: A ``bool`` value. If ``False``, the emoji will not
+            be available in the emoji selection for user flair.
+            (default: ``True``).
+        :param mod_only:  A ``bool`` value. If ``True``, the emoji will only
+            be available for use.
```

Comment:
I believe this should say "available for mod use," not just "available for use."

Reply:
Oops, yeah. Too much copy/pasting out of frustration.